tests/Doctrine/Tests/Common/Persistence/Mapping/SymfonyFileLocatorTest.php

```diff
@@ -62,6 +62,90 @@ public function testGetAllClassNames()
         $this->assertEquals(array("Foo\\stdClass"), $locator->getAllClassNames("global"));
     }
 
+
+    public function customNamespaceSeparatorProvider()
+    {
+        return array(
+            'directory separator' => array(DIRECTORY_SEPARATOR, "/_custom_ns/dir"),
+            'underscore separator' => array('_', "/_custom_ns/underscore"),
```

Comment:
this case is a broken setup, so the testsuite should not advocate it (see my comment for the explanation)